Infrastructure/Migrations/20220705165059_CoverageIndexes.Designer.cs

```diff
@@ -4181,7 +4181,7 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
 
                     b.Navigation("LocalAuthorityDistrict");
 
-                    b.Navigation("TuitionPartner");
+                    b.Navigation("ShortlistedTuitionPartner");
```

Comment:
Still quite a few "ShortlistedTuitionPartner" changes to various Migration Designer files that need reverting